annex/annexA.md

```diff
@@ -0,0 +1,19 @@
+ANNEX
+-----
+
+This annex is the follow-up of examples related to chapter 1 in this tutorial. 
+
+The example7.cpp is about: 
+- Listener to print the content of the event and the thread id
+- Publisher and subscriber for other types of data: int, my own struct such as Position(float _x, float _y)
+
+
+The example8.cpp is about: 
+- 1 event loop
+- Publisher and subscriber for other type of data: in this case Position(float _x, float _y)
+- Distance to origin is computed and displayed by the subscriber
+
+The example9.cpp is about:
```

Comment:
Same - explain why example 9 is included, what it is intended to show to the reader of the tutorial. Then explain what it does.

Reply:
Explanation added